apps/web/src/components/templates/in-app-editor/FeedItems.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState } from 'react';
+import { Popover, useMantineTheme, Grid, ColorScheme } from '@mantine/core';
+import { FeedChip } from './FeedChip';
+import { colors, shadows, Text, Tooltip, Button } from '../../../design-system';
+import { Copy, Trash } from '../../../design-system/icons';
+import { useClipboard } from '@mantine/hooks';
+import { FeedEntity } from '@novu/dal';
```

Comment:
@novu/dal import alert :P

Reply:
Will remove it shortly! thanks for the heads up